lib/susedistribution.pm

```diff
@@ -357,15 +362,16 @@ sub activate_console {
         }
     }
 
-    if ($console =~ m/^(.*)-console/) {
-        my ($name, $user) = ($1, $1);
-        if ($name eq 'user') {
-            $user = $testapi::username;
-        }
-        elsif ($name eq 'log') {
-            $user = 'root';
-        }
+    $console =~ m/^(\w+)-(console|virtio-terminal)/;
+    my ($name, $user, $type) = ($1, $1, $2);
```

Comment:
I think the solution is quite simply to do the following:
```my ($name, $user, $type) = ($1 || '', $1 || '', $2 || '');```

Once I have tested it I will make a pull request.